packages/testlab/test/integration/test-sandbox.integration.ts

```diff
@@ -118,7 +118,7 @@ describe('TestSandbox integration tests', () => {
   }
 
   async function deleteSandbox() {
-    if (!await pathExists(path)) return;
+    if (!(await pathExists(path))) return;
```

Comment:
This change is needed to fix the CI after a recent `prettier` version changed the formatting of `await` statements. It comes in a standalone commit.

Reply:
Please also upgrade to "prettier": "^1.12.1" so that the new code won't fail with existing checkout which still uses `prettier@1.11.x`.